push/Push.md

```diff
@@ -23,3 +23,17 @@ This is essential due to a couple of reasons:
 This is also applicable to product Docker images available at [WSO2 Private Docker Registry](https://docker.wso2.com) mainly due to limitations of storage. For each product version, new images are being frequently added with the latest [WSO2 Updates](https://wso2.com/updates) to products and changes to docker resources used. With limitations of storage, not all images being pushed could be retained in the registry for a long time.
 
 As a result, WSO2 best advise the use of your own private container registry to host all the required images in your deployments, so that you have full control over them. In a rollback event, the recovery of previous version images is effortless with zero risks and zero dependence on outside sources.
+
+## <a name="how-to-secure"></a>2.2. How to secure your private registry ?
+
+Your private registry is the place where you host all the images required for your deployment. When you push an image to the registry, it is always important to be sure that you push the exact image that you want to store in your registry. When you pull an image from the registry, it is also important to be sure that you pull the exact image that you want to retrieve from your registry. Since the transfer of data in both of these situations happens in HTTP, to avoid any man-in-the-middle attacks, securing the transfer using `SSL security` is essential. 
```

Comment:
@DilanUA shall we break the `Since the transfer of data in both of these situations happens in HTTP, to avoid any man-in-the-middle attacks, securing the transfer...` sentence into a separate paragraph or point?

Reply:
@chirangaalwis How about this? Since the transfer of data in both of these situations happens in HTTP, securing the transfer using `SSL security` is essential to avoid any man-in-the-middle attacks.

Reply:
done